package.json

```diff
@@ -50,7 +45,7 @@
     "webpack-dev-server": "^1.14.1"
   },
   "engines": {
-    "node": ">=0.10.0"
+    "node": ">=12"
```

Comment:
Is node 12 still fine with the eslint config you added?

Reply:
yes, https://github.com/livingdocsIO/editable.js/pull/204#discussion_r535994941

Reply:
we could also just upgrade to node 14, but then I'd remove the engines declaration in here, as it shouldn't be in the final module build. This library works in the browser, not on node, so it could report false negatives during an install of a project that depends on node 12. e.g. our projects.

Reply:
I think node 12 is fine for the moment.